src/seedu/addressbook/commands/PostalCode.java

```diff
@@ -0,0 +1,16 @@
+package seedu.addressbook.data.person;
+ 
+ public class PostalCode {
+ 	private final String postalCode;
+ 	
+ 	public PostalCode(String postalCode) {
+ 		this.postalCode = postalCode;
+ 	}
+ 
+ 	/**
```

Comment:
can eliminate a header comment that give no additional info than the method signature